akka-docs/src/test/java/jdocs/actor/FaultHandlingTest.java

```diff
@@ -57,11 +53,13 @@
 
     //#strategy
     private static SupervisorStrategy strategy =
-      new OneForOneStrategy(10, Duration.create(1, TimeUnit.MINUTES), DeciderBuilder.
-        match(ArithmeticException.class, e -> resume()).
-        match(NullPointerException.class, e -> restart()).
-        match(IllegalArgumentException.class, e -> stop()).
-        matchAny(o -> escalate()).build());
+      new OneForOneStrategy(10, Duration.create(1, TimeUnit.MINUTES),
+          DeciderBuilder
+              .match(ArithmeticException.class, e -> SupervisorStrategy.resume())
+              .match(NullPointerException.class, e -> SupervisorStrategy.restart())
+              .match(IllegalArgumentException.class, e -> SupervisorStrategy.stop())
```

Comment:
Isn't the `SupervisorStrategy` missing from the imports now?

Reply:
Goddamn ;) Yes, thanks